runtime-testsuite/resources/org/antlr/v4/test/runtime/templates/LexerExec/CharSet.stg

```diff
@@ -1,21 +1,17 @@
 TestType() ::= "Lexer"
 
-Options ::= [
-	"Ignore": "Currently unable to generate the required CRLF input sequence."
-]
-
 Grammar ::= [
 	"L": {<grammar("L")>}
 ]
 
-Input() ::= "34<\u000D><\n> 34"
```

Comment:
:exclamation: I'm planning to fix the handling of carriage returns to ensure the test suite includes coverage of these characters, which do not behave the same way as new line characters in ANTLR. Until it's working under the original design, I would prefer to not make any changes to this.